starlite/utils/module_loading.py

```diff
@@ -0,0 +1,32 @@
+import sys
+from importlib import import_module
+
+
+def import_string(dotted_path):
+    """
+    Import a dotted module path and return the attribute/class designated by the
+    last name in the path. Raise ImportError if the import failed.
+
+    Based on Django's module_loading code
+
+    """
+
+    def cached_import(module_path, class_name):
+        # Check whether module is loaded and fully initialized.
+        if not (
+            (module := sys.modules.get(module_path))
```

Comment:
walrus operator is not supported in python 3.7.

Reply:
also, its best to break this into multiple variables and be more descriptive. This code is not quite clear enough.